ci-operator/config/openshift-eng/ocp-qe-perfscale-ci/openshift-eng-ocp-qe-perfscale-ci-main__gcp-4.15-nightly-x86.yaml

```diff
@@ -16,7 +16,7 @@ resources:
       memory: 200Mi
 tests:
 - as: control-plane-etcd-fips-24nodes
-  cron: 0 10 9 * *
+  cron: 0 10 9 7 *
```

Comment:
"one final run in july"
We must remember to delete these after they run in july.